src/components/Navbar.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ]  @MasumaJaffery It's not good practice disabling a linter check in your file, linter checks help us write quality code, kindly consider removing this line